### PR TITLE
Update query test assertions

### PR DESCRIPTION
**Pull request by cynicaljoy**
## Problem

Some of the tests are too explicit

## Solution

- Only assert the `data` on the result
- replace `.not.toBeUndefined()` with `toBeDefined()`
- don't validate `summary` contents, just that it's set 

## Result

A few tests pass

## Out of scope

Fix remaining tests

### CODE REVIEW

File: __tests__/functional/client-configuration.test.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ an environmental variable named FAUNA_SECRET or pass it to the Client constructo
     expect(client.client.defaults.baseURL).toEqual("http://localhost:7443/");
     const result = await client.query<number>({ query: '"taco".length' });
     expect(result.txn_time).not.toBeUndefined();
-    expect(result).toEqual({ data: 4, txn_time: result.txn_time });
+    expect(result).toEqual({ data: 4, txn_time: result.txn_time, summary: "" });
```
Comment (cleve-fauna) [inline __tests__/functional/client-configuration.test.ts:63]: This seems like odd behavior from Core.

@freels @macmv is it intended that core return empty strings for summary when there's no summary data?

Comment (freels) [reply __tests__/functional/client-configuration.test.ts:63]: No, it probably shouldn’t do that. In general my philosophy too is that the drivers shouldn’t be testing core behavior like this, so if possible I’d tweak it to only check the data field itself or something.

Comment (cleve-fauna) [inline __tests__/functional/client-configuration.test.ts:63]:
```suggestion
    expect(result).toMatchObject({ data: 4, txn_time: result.txn_time });
```

Comment (cleve-fauna) [reply __tests__/functional/client-configuration.test.ts:63]: @cynicaljoy this will fix the test ^^^

Comment (cynicaljoy) [reply __tests__/functional/client-configuration.test.ts]: I posted a fix before I saw this, let me know if you still want me to use this pattern or not. if not, please just mark conversation resolved.

**Review by cleve-fauna — APPROVED** [review]

File: __tests__/functional/client-configuration.test.ts [hunk 2]
```diff
@@ -43,6 +43,8 @@ an environmental variable named FAUNA_SECRET or pass it to the Client constructo
   });
 
   it("endpoints is extensible", async () => {
+    expect.assertions(4);
```
Comment (cleve-fauna) [inline __tests__/functional/client-configuration.test.ts:46]: You don't need this for tests that don't catch errors.

Comment (cleve-fauna) [inline __tests__/functional/client-configuration.test.ts:46]:
```suggestion
```

**Review by cleve-fauna — CHANGES_REQUESTED** [review]

File: __tests__/functional/client-configuration.test.ts (hunk 2 above)
Comment (cleve-fauna) [reply __tests__/functional/client-configuration.test.ts:46]: these are not needed for tests taht don't assertions in a catch block.

**Review by cleve-fauna — APPROVED** [review]

**Pull request merged**